.editorconfig

```diff
@@ -0,0 +1,15 @@
+# EditorConfig <http://EditorConfig.org>
```

Comment:
i am not sure if this was intended but it seems not needed to be in the repo.

Reply:
Yes, @cassidyjames provide the official file.